dashboard_description_plugin/webpack.config.js

```diff
@@ -165,7 +165,7 @@ module.exports = (_env, argv) => {
             plugins: [
                 ...commonConfig.plugins,
                 new ProvidePlugin({
-                    process: "process/browser",
+                    process: "process/browser.js",
```

Comment:
Why is this change needed? Should we update the template for other plugins as well?

Reply:
This was needed due to the way`react-markdown` is packaged, it would not compile otherwise.
```
ERROR in ./node_modules/uvu/node_modules/kleur/index.mjs 4:11-18
  Module not found: Error: Can't resolve 'process/browser' in '.../gooddata-plugin-examples/dashboard_description_plugin/node_modules/uvu/node_modules/kleur'
  Did you mean 'browser.js'?
  BREAKING CHANGE: The request 'process/browser' failed to resolve only because it was resolved as fully specified
  (probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
  The extension in the request is mandatory for it to be fully specified.
  Add the extension to the request.
```

Reply:
Ok, thanks for the explanation :)